File: leetcode/two-sum/python/two_sum_brute_force.py
class Solution:
    def two_sum(self, nums: list[int], target: int) -> list[int]:
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[j] == target - nums[i]:
                    return [i, j]
        return []

sol = Solution()
print(sol.two_sum([2, 7, 11, 15], 9))

File: leetcode/two-sum/javascript/two-sum-brute-force.js
const twoSum = function(numbers, target) {
    for (let i = 0, l = numbers.length; i < l; i += 1) {
        for (let j = i + 1, m = numbers.length; j < m; j += 1) {
            if (numbers[j] === target - numbers[i]) {
                return [i, j];
            }
        }
    }
    return [];
};

console.log(twoSum([2, 4, 5, 6, 11], 10));